modules/components/rest_form_element.cjsx

```diff
@@ -19,32 +20,13 @@ module.exports = class RestFormElement extends React.Component
     value:                 React.PropTypes.any
 
 
-  constructor: (props) ->
-    @state =
-      value: props.value ? props.defaultValue ? ''
-
-
   componentWillMount: ->
     if @props.defaultValue and not @props.value
       @props.onChange @props.name, @props.defaultValue
```

Comment:
Thinking we should turf `defaultValue` altogether since the user can just set a default value in the `model`.